article/app/controllers/LiveBlogController.scala

```diff
@@ -61,16 +69,25 @@ class LiveBlogController(
           case (minute: MinutePage, HtmlFormat) =>
             Future.successful(common.renderHtml(MinuteHtmlPage.html(minute), minute))
           case (blog: LiveBlogPage, HtmlFormat) => {
-            // dcrCanRender is always false because right now blogs are not supported by DCR
+            // dcrCouldRender is always false because right now blogs are not supported by DCR
             // but we included this variable as an indication of what is going to be possible in the future
-            val dcrCanRender = false
+            val dcrCouldRender = false
             val participatingInTest = ActiveExperiments.isParticipating(LiveblogRendering)
+            val properties =
+              Map(
+                "participatingInTest" -> participatingInTest.toString(),
+                "dcrCouldRender" -> dcrCouldRender.toString(),
+              )
             val remoteRendering =
-              shouldRemoteRender(request.forceDCROff, request.forceDCR, participatingInTest, dcrCanRender)
+              shouldRemoteRender(request.forceDCROff, request.forceDCR, participatingInTest, dcrCouldRender)
 
             remoteRendering match {
-              case false => Future.successful(common.renderHtml(LiveBlogHtmlPage.html(blog), blog))
+              case false => {
+                logRequest(s"liveblog executing in web", properties, page)
+                Future.successful(common.renderHtml(LiveBlogHtmlPage.html(blog), blog))
+              }
               case true => {
+                logRequest(s"liveblog executing in dotcomponents", properties, page)
```

Comment:
does this duplicate the existing logging for DCR? https://github.com/guardian/frontend/blob/4dabb0b00619d84779fe16c8efcfbcfd804554a4/article/app/services/dotcomponents/ArticlePicker.scala#L101

Reply:
I thought `ArticlePicker` was only for articles, not for liveblogs - so that code won't execute if you're a liveblog and end up in the `LiveBlogController`?